File: oss-manage/modules/common/responseData.js
module.exports = ResponseData;

ResponseData.prototype.code;
ResponseData.prototype.msg;
ResponseData.prototype.data;

function ResponseData() {
    //构造方法内的属性和方法
}

function ResponseData(code, msg) {
    //构造方法内的属性和方法
    this.code = code;
    this.msg = msg;
}

function ResponseData(code, msg, data) {
    //构造方法内的属性和方法
    this.code = code;
    this.msg = msg;
    this.data = data;
}

ResponseData.prototype.ResponseData = function (code, msg) {
    //构造方法内的属性和方法
    this.code = code;
    this.msg = msg;
}

ResponseData.prototype.ResponseData = function (code, msg, data) {
    //构造方法内的属性和方法
    this.code = code;
    this.msg = msg;
    this.data = data;
}

File: commonModule/src/main/java/pres/gogym/common/ResponseData.java
/*
 * 文件名：ResponseData.java
 * 版权：Copyright by www.poly.com
 * 描述：
 * 修改人：gogym
 * 修改时间：2019年3月12日
 * 跟踪单号：
 * 修改单号：
 * 修改内容：
 */

package pres.gogym.common;

public class ResponseData<T> {

	private Integer code;
	private String msg;
	private T data;

	public ResponseData() {
	}

	public ResponseData(Integer code, String msg) {
		this.code = code;
		this.msg = msg;
	}

	public ResponseData(Integer code, String msg, T data) {
		this.code = code;
		this.msg = msg;
		this.data = data;
	}

}
